content/en/docs/instrumentation/cpp/getting-started-new.md

```diff
@@ -0,0 +1,388 @@
+---
+title: Getting Started
+description: Get telemetry for your app in less than 5 minutes!
+cSpell:ignore: rolldice
+weight: 10
+---
+
+This page will show you how to get started with OpenTelemetry in C++.
+
+You will learn how to instrument a simple C++ application automatically,
+such that [traces](/docs/concepts/signals/traces/), [metrics](/docs/concepts/signals/metrics/) and [logs](/docs/concepts/signals/logs/) are emitted to the terminal.
+
+## Prerequisites 
+
+Ensure that you have the following installed locally:
+
+- Git
+- C++ compiler supporting C++ version >= 14
+- Make 
+- CMake version >= 3.1
+
+## Example Application 
+
+The following example uses a basic [Oat++](https://oatpp.io/) application. If you are 
+not using Oat++, that's OK - you can use OpenTelemetry C++ with any other web frameworks as well.
+
+## Dependencies 
+
+**To begin, we will install Oatpp locally using the [source code](https://github.com/oatpp) and `make`, following these general steps:**
+
+1. Download oatpp source code:
+
+Obtain the oatpp source code. You can either download the source code from the oatpp Github repository or clone it using Git.
```

Comment:
Indent the content of list entries, like this line (line 33), otherwise the list entry content won't display properly:

> <img width="1040" alt="image" src="https://github.com/open-telemetry/opentelemetry.io/assets/4140793/90f9ae0c-dc41-4c2b-84ba-56a1abeb9215">